www/front_src/src/Resources/Graph/Performance/timeSeries.ts

```diff
@@ -56,13 +56,15 @@ const getTimeSeries = (graphData: GraphData): Array<MetricData> => {
   )(graphData);
 };
 
-interface LegendColor {
-  legend: string;
+export interface LegendColor {
```

Comment:
This interface seems only used locally
```suggestion
interface LegendColor {
```